src/pybind/mgr/dashboard/frontend/src/app/shared/components/modal/modal.component.scss

```
@@ -5,12 +5,12 @@
   border-radius: 5px 5px 0 0;
 }
 
-::ng-deep .modal-footer {
+::ng-deep cd-modal.modal-footer {
   @include mixins.hf;
   border-radius: 0 0 5px 5px;
 }
 
-::ng-deep .modal-body {
+::ng-deep cd-modal.modal-body {
```

This should go inside the previous `::ng-deep`
```suggestion
.modal-body {
```